### DISCUSSION TITLE
Ability to debug scripting Filters?

### DISCUSSION THREAD

**Discussion by CharlieSnowCode** [thunderclient/thunder-client-support#1024] [Q&A] [answered]
I have a Collection that utilizes a JavaScript file - which is working.  I use it for some custom Tests we are writing.  I can set a breakpoint in the script but can't figure out how to debug.  Is this a supported feature?  
Many thanks.

**Comment by rangav** [comment, marked as answer]
Hi @CharlieSnowCode breakpoint and debug is not possible i think. But you can use console.log and see in the logs view

<img width="779" alt="Screenshot 2023-02-22 at 21 04 37" src="https://user-images.githubusercontent.com/8637550/220759015-e9a1251a-0456-4eb9-8179-9d5de84935d5.png">

**Reply by CharlieSnowCode** [reply]
Okay, thanks for the quick reply.
